### ISSUE TITLE
Flush get lower performance

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#1615]
**Describe the bug**

```
2020-03-11 09:44:26,569 | DEBUG | default | [SERVER] DeleteByIDRequest: check validation (0.000121 second [0.120823 ms])
2020-03-11 09:44:26,569 | DEBUG | default | [SERVER] DeleteByIDRequest: totally cost (0.000259 second [0.258762 ms])
2020-03-11 09:44:26,570 | DEBUG | default | [SERVER] FlushRequest flush tables: sift_10m_1024_128_l2,
2020-03-11 09:44:26,570 | DEBUG | default | [ENGINE] Begin flush table: sift_10m_1024_128_l2
2020-03-11 09:44:26,570 | DEBUG | default | [ENGINE] WAL flush
2020-03-11 09:44:26,570 | DEBUG | default | [ENGINE] wal_mgr_->Flush
2020-03-11 09:44:26,975 | DEBUG | default | [ENGINE] Flushing table: sift_10m_1024_128_l2
2020-03-11 09:44:26,975 | DEBUG | default | [ENGINE] Applying 5000 deletes in table: sift_10m_1024_128_l2
2020-03-11 09:44:26,976 | DEBUG | default | [ENGINE] Get table files by type. raw files:0 to_index files:0 backup files:5
2020-03-11 09:44:26,976 | DEBUG | default | [ENGINE] Mark ongoing file:1583586239353258000 refcount:1
2020-03-11 09:44:26,976 | DEBUG | default | [ENGINE] Mark ongoing file:1583586495355089000 refcount:1
2020-03-11 09:44:26,976 | DEBUG | default | [ENGINE] Mark ongoing file:1583586748580797000 refcount:1
2020-03-11 09:44:26,976 | DEBUG | default | [ENGINE] Mark ongoing file:1583586927098464000 refcount:1
2020-03-11 09:44:26,977 | DEBUG | default | [ENGINE] Mark ongoing file:1583587017825755000 refcount:1
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586239353258000 refcount:0
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586495355089000 refcount:0
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586748580797000 refcount:0
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586927098464000 refcount:0
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Unmark ongoing file:1583587017825755000 refcount:0
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Mark ongoing file:1583587017825755000 refcount:1
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Mark ongoing file:1583586239353258000 refcount:1
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Mark ongoing file:1583586495355089000 refcount:1
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Mark ongoing file:1583586748580797000 refcount:1
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Mark ongoing file:1583586927098464000 refcount:1
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Found 5 segment to apply deletes in 0.666154 s
2020-03-11 09:44:27,641 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583587017825755000
2020-03-11 09:44:27,642 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-11 09:44:27,673 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0316938 s
2020-03-11 09:44:27,673 | DEBUG | default | [ENGINE] Sorting 32 ids took 2.46e-06 s
2020-03-11 09:44:29,007 | DEBUG | default | [ENGINE] Finding 32 uids in 1437856 uids took 0.66886 s in total
2020-03-11 09:44:29,007 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-11 09:44:29,216 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583587017825755000 in 0.208801 s
2020-03-11 09:44:29,219 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583587017825755000 in 0.00307861 s
2020-03-11 09:44:29,219 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586239353258000
2020-03-11 09:44:29,220 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-11 09:44:29,257 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0378816 s
2020-03-11 09:44:29,257 | DEBUG | default | [ENGINE] Sorting 52 ids took 3.326e-06 s
2020-03-11 09:44:31,187 | DEBUG | default | [ENGINE] Finding 52 uids in 2100000 uids took 0.969457 s in total
2020-03-11 09:44:31,187 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-11 09:44:31,249 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586239353258000 in 0.0621873 s
2020-03-11 09:44:31,251 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586239353258000 in 0.00246549 s
2020-03-11 09:44:31,252 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586495355089000
2020-03-11 09:44:31,253 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-11 09:44:31,351 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0990413 s
2020-03-11 09:44:31,351 | DEBUG | default | [ENGINE] Sorting 44 ids took 3.203e-06 s
2020-03-11 09:44:33,273 | DEBUG | default | [ENGINE] Finding 44 uids in 2100000 uids took 0.965426 s in total
2020-03-11 09:44:33,273 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-11 09:44:33,318 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586495355089000 in 0.0443432 s
2020-03-11 09:44:33,321 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586495355089000 in 0.00273614 s
2020-03-11 09:44:33,321 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586748580797000
2020-03-11 09:44:33,322 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-11 09:44:33,354 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0333821 s
2020-03-11 09:44:33,354 | DEBUG | default | [ENGINE] Sorting 34 ids took 2.668e-06 s
2020-03-11 09:44:35,425 | DEBUG | default | [ENGINE] Finding 34 uids in 2262144 uids took 1.04033 s in total
2020-03-11 09:44:35,425 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-11 09:44:35,444 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586748580797000 in 0.0183628 s
2020-03-11 09:44:35,446 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586748580797000 in 0.00220637 s
2020-03-11 09:44:35,446 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586927098464000
2020-03-11 09:44:35,447 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-11 09:44:35,476 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0292757 s
2020-03-11 09:44:35,476 | DEBUG | default | [ENGINE] Sorting 54 ids took 2.88e-06 s
2020-03-11 09:44:37,405 | DEBUG | default | [ENGINE] Finding 54 uids in 2100000 uids took 0.971415 s in total
2020-03-11 09:44:37,405 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-11 09:44:37,438 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586927098464000 in 0.033056 s
2020-03-11 09:44:37,440 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586927098464000 in 0.00249547 s
2020-03-11 09:44:37,441 | DEBUG | default | [ENGINE] Update 10 table files
2020-03-11 09:44:37,442 | DEBUG | default | [ENGINE] Finished applying deletes in table sift_10m_1024_128_l2 in 10.4669 s
2020-03-11 09:44:37,442 | DEBUG | default | [ENGINE] Unmark ongoing file:1583587017825755000 refcount:0
2020-03-11 09:44:37,442 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586239353258000 refcount:0
2020-03-11 09:44:37,442 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586495355089000 refcount:0
2020-03-11 09:44:37,442 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586748580797000 refcount:0
2020-03-11 09:44:37,442 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586927098464000 refcount:0
2020-03-11 09:44:37,444 | DEBUG | default | [ENGINE] Initializing bloom filter took 0.00166653 s
2020-03-11 09:44:37,488 | DEBUG | default | [ENGINE] Adding 95000 ids to bloom filter took 0.0441896 s
2020-03-11 09:44:37,488 | DEBUG | default | [ENGINE] Writing bloom filter took 2.737e-06 s
2020-03-11 09:44:37,488 | DEBUG | default | [ENGINE] Writing bloom filter took 0.046341 s in total
2020-03-11 09:44:37,578 | DEBUG | default | [ENGINE] Writing raw vectors took 0.0878204 s
2020-03-11 09:44:37,580 | DEBUG | default | [ENGINE] Writing uids took 0.00214428 s
2020-03-11 09:44:37,580 | DEBUG | default | [ENGINE] Writing vectors and uids took 0.0914014 s in total
2020-03-11 09:44:37,582 | DEBUG | default | [ENGINE] Writing deleted docs took 0.00188737 s
2020-03-11 09:44:37,582 | DEBUG | default | [ENGINE] Update single table file, file id = 1583891066566806000
2020-03-11 09:44:37,582 | DEBUG | default | [ENGINE] New raw file 1583891066566806000 of size 51200000 bytes, lsn = 167555764626
```
**Steps/Code to reproduce behavior**

dataset based: sift10m-128d
index_type: ivf_sq8

operations:

1. insert 100,000 vectors;
2. delete 5,000 vectors;
3. flush
4. compact

**Expected behavior**

**Environment details**
gpu-verison
commit id: 5b0b226bc42293a39b79b7317b3aeba50f96b78e

**Screenshots**

**Additional context**

**Comment by youny626** [comment]
Deleting 5000 out of 10 million vectors takes around 3 or 4 seconds. With NFS the time would double.
However, deleting 1 million out of 10 million vectors only takes around 5 or 6 seconds. The time really depends on the data size, and how the deletions are distributed across different segments since bloom filter will rule out some. 
Bottom line is this is normal, we might have some space left for improvement, but it's def not a bug.

**Comment by del-zhenwu** [comment]
flush time cost:
```
22:33:55  2020-03-23:14:33:51,930 INFO     [client.py:33] Milvus flush run in 15.73s
22:34:22  2020-03-23:14:34:18,896 INFO     [client.py:33] Milvus flush run in 19.03s
22:34:53  2020-03-23:14:34:50,300 INFO     [client.py:33] Milvus flush run in 20.93s
22:35:28  2020-03-23:14:35:27,226 INFO     [client.py:33] Milvus flush run in 26.44s
...
22:44:19  2020-03-23:14:44:13,735 INFO     [client.py:33] Milvus flush run in 41.16s
22:45:18  2020-03-23:14:45:15,781 INFO     [client.py:33] Milvus flush run in 45.21s
22:47:07  2020-03-23:14:47:04,459 INFO     [client.py:33] Milvus flush run in 94.15s
...
```
more info need check server logs in : ```
/test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/logs/milvus-20-03-23-22:32-debug.log
```

**Comment by youny626** [comment]
Unable to reproduce

**Comment by del-zhenwu** [comment]
> Unable to reproduce

```
operations:

insert 100,000 vectors;
delete 5,000 vectors;
flush
compact
```
loop 10 times, flush time increase each time

**Comment by youny626** [comment]
> > Unable to reproduce
> 
> ```
> operations:
> 
> insert 100,000 vectors;
> delete 5,000 vectors;
> flush
> compact
> ```
> 
> loop 10 times, flush time increase for each time

Tested the above operations, looping 100 times. Still unable to reproduce. Tested on NAS in DEBUG mode, flush took at most 20 seconds.

**Comment by youny626** [comment]
2020-03-25 13:53:35,474 | DEBUG | default | [SERVER] FlushRequest flush tables: sift_10m_1024_128_l2, 
2020-03-25 13:53:35,475 | DEBUG | default | [ENGINE] Begin flush table: sift_10m_1024_128_l2
2020-03-25 13:53:35,475 | DEBUG | default | [ENGINE] WAL flush
2020-03-25 13:53:35,475 | DEBUG | default | [ENGINE] wal_mgr_->Flush
2020-03-25 13:54:20,648 | DEBUG | default | [SERVER] write vectors: write rv done (47.850348 seconds [47850.347838 ms])
2020-03-25 13:54:21,337 | DEBUG | default | [SERVER] write vectors: write uids done (0.689326 second [689.326316 ms])
**2020-03-25 13:54:21,337 | DEBUG | default | [ENGINE] Writing vectors and uids took 48.5412 s in total**
2020-03-25 13:54:21,341 | DEBUG | default | [ENGINE] Writing deleted docs took 0.00367431 s
2020-03-25 13:54:21,342 | DEBUG | default | [ENGINE] Update 3 table files
2020-03-25 13:54:21,342 | DEBUG | default | [ENGINE] New merged segment 1585115606462777000 of size 1086800000 bytes
2020-03-25 13:54:21,435 | DEBUG | default | [ENGINE] Unmark ongoing file:1585115537992546000 refcount:0
2020-03-25 13:54:21,435 | DEBUG | default | [ENGINE] Unmark ongoing file:1585115591007347000 refcount:0
2020-03-25 13:54:21,439 | DEBUG | default | [SERVER] Erase /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115472069658000/1585115472069658000 size: 988237500 bytes from cache, usage: 6428373535 bytes, capacity: 68719476736 bytes
2020-03-25 13:54:21,506 | DEBUG | default | [ENGINE] Remove file id:1585115472069658000 location:/test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115472069658000/1585115472069658000
2020-03-25 13:54:21,506 | DEBUG | default | [SERVER] Erase /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115523063627000/1585115523063627000 size: 49411875 bytes from cache, usage: 6378961660 bytes, capacity: 68719476736 bytes
2020-03-25 13:54:21,513 | DEBUG | default | [ENGINE] Remove file id:1585115523063627000 location:/test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115523063627000/1585115523063627000
2020-03-25 13:54:21,513 | DEBUG | default | [ENGINE] Clean 2 files expired in 10 seconds
2020-03-25 13:54:21,555 | DEBUG | default | [ENGINE] Collect 1 to-index files
2020-03-25 13:54:21,555 | DEBUG | default | [ENGINE] Background build index thread begin
2020-03-25 13:54:21,555 | DEBUG | default | [ENGINE] Mark ongoing file:1585115606462777000 refcount:1
2020-03-25 13:54:21,555 | DEBUG | default | [SERVER] BuildIndexJob 92 add to_index file: 151, location: /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115606462777000
2020-03-25 13:54:21,555 | DEBUG | default | [ENGINE] EngineFactory index type: 1
2020-03-25 13:54:21,555 | DEBUG | default | [ENGINE] Index params: {"dim":128,"gpu_id":0,"metric_type":"L2","nlist":4096}
2020-03-25 13:54:21,555 | DEBUG | default | [SERVER] FallbackPass!
2020-03-25 13:54:21,555 | DEBUG | default | [SERVER] cpu load BuildIndexTask
2020-03-25 13:54:21,555 | DEBUG | default | [ENGINE] Index params: {"dim":128,"gpu_id":0,"metric_type":"L2"}
2020-03-25 13:54:21,612 | DEBUG | default | [ENGINE] Remove segment directory: /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115472069658000
2020-03-25 13:54:21,628 | DEBUG | default | [ENGINE] Remove segment directory: /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115523063627000
2020-03-25 13:54:21,628 | DEBUG | default | [ENGINE] Remove 2 segments folder
2020-03-25 13:54:21,629 | DEBUG | default | [ENGINE] Successfully create table file, file id = 1585115661629087000
2020-03-25 13:54:22,016 | DEBUG | default | [ENGINE] Flushing table: sift_10m_1024_128_l2
2020-03-25 13:54:22,016 | DEBUG | default | [ENGINE] Applying 5000 deletes in table: sift_10m_1024_128_l2
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Get table files by type. raw files:0 to_index files:1 backup files:5
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Mark ongoing file:1583586239353258000 refcount:1
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Mark ongoing file:1583586495355089000 refcount:1
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Mark ongoing file:1583586748580797000 refcount:1
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Mark ongoing file:1583586927098464000 refcount:1
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Mark ongoing file:1583587017825755000 refcount:1
2020-03-25 13:54:22,018 | DEBUG | default | [ENGINE] Mark ongoing file:1585115606462777000 refcount:2
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586239353258000 refcount:0
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586495355089000 refcount:0
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586748580797000 refcount:0
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586927098464000 refcount:0
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Unmark ongoing file:1583587017825755000 refcount:0
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Unmark ongoing file:1585115606462777000 refcount:1
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Mark ongoing file:1585115606462777000 refcount:2
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Mark ongoing file:1583587017825755000 refcount:1
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Mark ongoing file:1583586239353258000 refcount:1
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Mark ongoing file:1583586495355089000 refcount:1
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Mark ongoing file:1583586748580797000 refcount:1
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Mark ongoing file:1583586927098464000 refcount:1
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Found 6 segment to apply deletes in 0.687553 s
2020-03-25 13:54:22,703 | DEBUG | default | [ENGINE] Applying deletes in segment: 1585115606462777000
2020-03-25 13:54:22,704 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-25 13:54:22,797 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0932684 s
2020-03-25 13:54:22,797 | DEBUG | default | [ENGINE] Sorting 45 ids took 9.2088e-05 s
2020-03-25 13:54:24,826 | DEBUG | default | [ENGINE] Finding 45 uids in 2090000 uids took 1.02029 s in total
2020-03-25 13:54:24,826 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-25 13:54:24,844 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1585115606462777000 in 0.0174163 s
2020-03-25 13:54:24,847 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1585115606462777000 in 0.00322456 s
2020-03-25 13:54:24,847 | DEBUG | default | [ENGINE] Update table file row count in vector of segment: 1585115606462777000 in 0.00322456 s
2020-03-25 13:54:24,847 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583587017825755000
2020-03-25 13:54:24,848 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-25 13:54:24,891 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0431849 s
2020-03-25 13:54:24,891 | DEBUG | default | [ENGINE] Sorting 37 ids took 6.9913e-05 s
2020-03-25 13:54:24,896 | DEBUG | default | [ENGINE] set uids 2090000 for index /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115606462777000
2020-03-25 13:54:26,291 | DEBUG | default | [ENGINE] Finding 37 uids in 1437856 uids took 0.70166 s in total
2020-03-25 13:54:26,291 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-25 13:54:26,319 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583587017825755000 in 0.0278355 s
2020-03-25 13:54:26,322 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583587017825755000 in 0.0035264 s
2020-03-25 13:54:26,322 | DEBUG | default | [ENGINE] Update table file row count in vector of segment: 1583587017825755000 in 0.0035264 s
2020-03-25 13:54:26,323 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586239353258000
2020-03-25 13:54:26,324 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-25 13:54:26,346 | DEBUG | default | [ENGINE] Finished loading raw data from segment /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000
2020-03-25 13:54:26,437 | DEBUG | default | [SERVER] Insert /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115606462777000 size: 1087061250 bytes into cache, usage: 7466022910 bytes, capacity: 68719476736 bytes
2020-03-25 13:54:26,437 | DEBUG | default | [SERVER] Build index task load file id:151 DISK2CPU file type:2 size:1087061250 bytes from location: /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115606462777000 totally cost (4.881308 seconds [4881.307990 ms])
2020-03-25 13:54:26,437 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.113992 s
2020-03-25 13:54:26,437 | DEBUG | default | [ENGINE] Sorting 44 ids took 6.887e-05 s
2020-03-25 13:54:26,438 | DEBUG | default | [ENGINE] Successfully create table file, file id = 1585115666437757000
2020-03-25 13:54:26,439 | DEBUG | default | [ENGINE] Begin build index for file:/test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115666437757000
2020-03-25 13:54:26,439 | DEBUG | default | [ENGINE] Build index file: /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115666437757000 from: /test/milvus/db_data_7/sift_10m_1024_128_l2_sq8_4096_wal_stability/db/tables/sift_10m_1024_128_l2/1585115606462777000/1585115606462777000
2020-03-25 13:54:26,439 | DEBUG | default | [ENGINE] Index params: {"dim":128,"gpu_id":0,"metric_type":"L2","nlist":4096,"rows":2090000}
2020-03-25 13:54:26,439 | DEBUG | default | [ENGINE] Index config: {"dim":128,"gpu_id":0,"metric_type":"L2","nlist":4096,"rows":2090000}
2020-03-25 13:54:28,571 | DEBUG | default | [ENGINE] Finding 44 uids in 2100000 uids took 1.07671 s in total
2020-03-25 13:54:28,571 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-25 13:54:28,601 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586239353258000 in 0.030516 s
2020-03-25 13:54:28,605 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586239353258000 in 0.00430581 s
2020-03-25 13:54:28,605 | DEBUG | default | [ENGINE] Update table file row count in vector of segment: 1583586239353258000 in 0.00430581 s
2020-03-25 13:54:28,606 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586495355089000
2020-03-25 13:54:28,606 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-25 13:54:28,677 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0708887 s
2020-03-25 13:54:28,677 | DEBUG | default | [ENGINE] Sorting 53 ids took 7.5665e-05 s
2020-03-25 13:54:30,995 | DEBUG | default | [ENGINE] Finding 53 uids in 2100000 uids took 1.16112 s in total
2020-03-25 13:54:30,995 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-25 13:54:31,015 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586495355089000 in 0.020039 s
2020-03-25 13:54:31,020 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586495355089000 in 0.00464725 s
2020-03-25 13:54:31,020 | DEBUG | default | [ENGINE] Update table file row count in vector of segment: 1583586495355089000 in 0.00464725 s
2020-03-25 13:54:31,020 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586748580797000
2020-03-25 13:54:31,021 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-25 13:54:31,062 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.0414001 s
2020-03-25 13:54:31,062 | DEBUG | default | [ENGINE] Sorting 56 ids took 6.0046e-05 s
2020-03-25 13:54:33,448 | DEBUG | default | [ENGINE] Finding 56 uids in 2262144 uids took 1.20358 s in total
2020-03-25 13:54:33,448 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-25 13:54:33,473 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586748580797000 in 0.0249295 s
2020-03-25 13:54:33,482 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586748580797000 in 0.00871491 s
2020-03-25 13:54:33,482 | DEBUG | default | [ENGINE] Update table file row count in vector of segment: 1583586748580797000 in 0.00871491 s
2020-03-25 13:54:33,483 | DEBUG | default | [ENGINE] Applying deletes in segment: 1583586927098464000
2020-03-25 13:54:33,484 | DEBUG | default | [ENGINE] Get table files by segment id
2020-03-25 13:54:33,554 | DEBUG | default | [ENGINE] Loading uids and deleted docs took 0.07101 s
2020-03-25 13:54:33,554 | DEBUG | default | [ENGINE] Sorting 49 ids took 6.618e-05 s
2020-03-25 13:54:35,783 | DEBUG | default | [ENGINE] Finding 49 uids in 2100000 uids took 1.11591 s in total
2020-03-25 13:54:35,783 | DEBUG | default | [ENGINE] Setting deleted docs and bloom filter took 0 s in total
2020-03-25 13:54:35,820 | DEBUG | default | [ENGINE] Appended 0 offsets to deleted docs in segment: 1583586927098464000 in 0.0371368 s
2020-03-25 13:54:35,826 | DEBUG | default | [ENGINE] Updated bloom filter in segment: 1583586927098464000 in 0.00596443 s
2020-03-25 13:54:35,826 | DEBUG | default | [ENGINE] Update table file row count in vector of segment: 1583586927098464000 in 0.00596443 s
2020-03-25 13:54:35,827 | DEBUG | default | [ENGINE] Update file 1585115606462777000 row count to 2090000
2020-03-25 13:54:35,828 | DEBUG | default | [ENGINE] Update file 1583587017825755000 row count to 1437856
2020-03-25 13:54:35,829 | DEBUG | default | [ENGINE] Update file 1585070239120101000 row count to 1437856
2020-03-25 13:54:35,830 | DEBUG | default | [ENGINE] Update file 1583586239353258000 row count to 2100000
2020-03-25 13:54:35,830 | DEBUG | default | [ENGINE] Update file 1585070186077614000 row count to 2100000
2020-03-25 13:54:35,831 | DEBUG | default | [ENGINE] Update file 1583586495355089000 row count to 2100000
2020-03-25 13:54:35,831 | DEBUG | default | [ENGINE] Update file 1585070198675829000 row count to 2100000
2020-03-25 13:54:35,832 | DEBUG | default | [ENGINE] Update file 1583586748580797000 row count to 2262144
2020-03-25 13:54:35,833 | DEBUG | default | [ENGINE] Update file 1585070214365153000 row count to 2262144
2020-03-25 13:54:35,833 | DEBUG | default | [ENGINE] Update file 1583586927098464000 row count to 2100000
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Update file 1585070226863411000 row count to 2100000
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Update deletes to meta in table sift_10m_1024_128_l2 in 0.00766075 s
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Finished applying deletes in table sift_10m_1024_128_l2 in 13.8185 s
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Unmark ongoing file:1585115606462777000 refcount:1
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Unmark ongoing file:1583587017825755000 refcount:0
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586239353258000 refcount:0
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586495355089000 refcount:0
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586748580797000 refcount:0
2020-03-25 13:54:35,834 | DEBUG | default | [ENGINE] Unmark ongoing file:1583586927098464000 refcount:0
2020-03-25 13:54:35,837 | DEBUG | default | [ENGINE] Initializing bloom filter took 0.00241117 s
2020-03-25 13:54:35,891 | DEBUG | default | [ENGINE] Adding 95000 ids to bloom filter took 0.0543327 s
2020-03-25 13:54:35,892 | DEBUG | default | [ENGINE] Writing bloom filter took 4.655e-06 s
2020-03-25 13:54:35,892 | DEBUG | default | [ENGINE] Writing bloom filter took 0.0572853 s in total
2020-03-25 13:54:37,857 | DEBUG | default | [SERVER] write vectors: write rv done (1.965589 seconds [1965.589385 ms])
2020-03-25 13:54:37,892 | DEBUG | default | [SERVER] write vectors: write uids done (0.035009 second [35.009387 ms])
2020-03-25 13:54:37,892 | DEBUG | default | [ENGINE] Writing vectors and uids took 2.00067 s in total
2020-03-25 13:54:37,897 | DEBUG | default | [ENGINE] Writing deleted docs took 0.00466978 s
2020-03-25 13:54:37,898 | DEBUG | default | [ENGINE] Update single table file, file id = 1585115661629087000
2020-03-25 13:54:37,898 | DEBUG | default | [ENGINE] New raw file 1585115661629087000 of size 51200000 bytes, lsn = 240574243380
2020-03-25 13:54:37,898 | DEBUG | default | [ENGINE] Flushed segment 1585115661629087000
2020-03-25 13:54:38,026 | DEBUG | default | [ENGINE] Successfully update table flush_lsn, table id = sift_10m_1024_128_l2
2020-03-25 13:54:38,026 | DEBUG | default | [ENGINE] Finished flushing for table sift_10m_1024_128_l2 in 16.0106 s
2020-03-25 13:54:38,026 | DEBUG | default | [ENGINE] Flushed table: sift_10m_1024_128_l2
2020-03-25 13:54:38,027 | DEBUG | default | [ENGINE] flush_task_swn_.Wait()
2020-03-25 13:54:38,027 | DEBUG | default | [ENGINE] End flush table: sift_10m_1024_128_l2
2020-03-25 13:54:38,027 | DEBUG | default | [SERVER] FlushRequest flush tables: sift_10m_1024_128_l2, : totally cost (62.552334 seconds [62552.334009 ms])

**Comment by del-zhenwu** [comment]
Maybe only occurs on given host, it test ok if changed to another machine
